All Components: Web editorial updates (#10899)
OfficeDev/office-ui-fabric-react: packages/office-ui-fabric-react/src/components/Rating/docs/new/RatingOverview.md

> Ratings show people’s opinions of a product, helping people make more informed purchasing decisions. People can also rate products they’ve purchased.

Reviewer: Ratings can be used for things other than products and purchasing decisions. An example that comes to mind is rating the quality of content, but I know there are others.

> Ratings show people's opinions of a product, helping people make more informed purchasing decisions. People can also rate products they've purchased.

Reviewer 2: ```suggestion
Ratings show people's opinions of a product they've purchased, helping other people make more informed purchasing decisions.
```

> [Sketch toolkit]()

Reviewer 2: These are supposed to have links right?

Reviewer: There aren't public individual component links available yet. They'll be added later before these docs go live (they're not going live immediately).
